Makes expected color codes of bug-loc test results consistent with current void-packages python3-Pygments version.
--- a/test/data/bug-loc.right
+++ b/test/data/bug-loc.right
@@ -1,26 +1,26 @@
 (bug-loc.sh:5):
-5:	[31mdirname[39;49;00m=[33m${[39;49;00m[31mBASH_SOURCE[39;49;00m%/*[33m}[39;49;00m   [37m# equivalent to dirname($0)[39;49;00m
+5:	[31mdirname[39;49;00m=[33m${[39;49;00m[31mBASH_SOURCE[39;49;00m%/*[33m}[39;49;00m[37m   [39;49;00m[37m# equivalent to dirname($0)[39;49;00m
 +# Test to see that we read in files that mentioned in breakpoints
 +# but we don't step into.
-+step
++step 
 (bug-loc.sh:6):
-6:	[36msource[39;49;00m [33m${[39;49;00m[31mdirname[39;49;00m[33m}[39;49;00m/library.sh
-+step
+6:	[36msource[39;49;00m[37m [39;49;00m[33m${[39;49;00m[31mdirname[39;49;00m[33m}[39;49;00m/library.sh[37m[39;49;00m
++step 
 (bug-loc.sh:7):
-7:	[36mecho[39;49;00m [33m'script line 7'[39;49;00m
-+step
+7:	[36mecho[39;49;00m[37m [39;49;00m[33m'script line 7'[39;49;00m[37m[39;49;00m
++step 
 script line 7
 (bug-loc.sh:8):
-8:	library-function
-+step
+8:	library-function[37m[39;49;00m
++step 
 (library.sh:1):
-1:	library-function() {
-+step
+1:	library-function()[37m [39;49;00m{[37m[39;49;00m
++step 
 (library.sh:2):
-2:	  [36mecho[39;49;00m [33m'library line 2 in library-function'[39;49;00m
-+step
+2:	[37m  [39;49;00m[36mecho[39;49;00m[37m [39;49;00m[33m'library line 2 in library-function'[39;49;00m[37m[39;49;00m
++step 
 library line 2 in library-function
 (bug-loc.sh:11):
-11:	[36mecho[39;49;00m [33m'script line 11'[39;49;00m
-+quit
+11:	[36mecho[39;49;00m[37m [39;49;00m[33m'script line 11'[39;49;00m[37m[39;49;00m
++quit 
 bashdb: That's all, folks...
